utils/firstTwoBytesOfAddress.ts

```diff
@@ -0,0 +1,7 @@
+const firstTwoBytesOfAddress = (address: string) => {
```

Comment:
maybe not worth using a single file for this very little function.
Maybe just create an utils.ts file so you can put a few more functions?